proxy/src/main/java/com/wavefront/agent/PushAgent.java

```diff
@@ -155,11 +155,11 @@ protected void startListeners() {
       }
     }
 
-    if (filebeatPort != null) {
+    if (filebeatPort > 0) {
       final Server filebeatServer = new Server(filebeatPort);
       filebeatServer.setMessageListener(new FilebeatListener(
           new PointHandlerImpl(filebeatPort, pushValidationLevel, pushBlockedSamples, getFlushTasks(filebeatPort)),
-          logsIngestionConfig, hostname, prefix, true));
+          logsIngestionConfig, hostname, prefix));
```

Comment:
if filebeatPort is > 0 and logsIngestionConfig is null, it will crash with NPE at start-up and from the exception text it's not obvious how can it be fixed - so it might be a good idea to check if both variables are set in the config file